tests/functional/export_v2/cram/auspice_config5.t

```diff
@@ -0,0 +1,16 @@
+Setup
+
+  $ source "$TESTDIR"/_setup.sh
+
+Run export with metadata and auspice config that include deprecated field names.
+
+  $ ${AUGUR} export v2 \
+  >   --tree "$TESTDIR/../data/tree.nwk" \
+  >   --node-data "$TESTDIR/../data/div_node-data.json" \
+  >   --auspice-config "$TESTDIR/../data/auspice_config5.json" \
+  >   --metadata "$TESTDIR/../data/deprecated_metadata.tsv" \
+  >   --output dataset.json &> /dev/null
+
+  $ python3 "$TESTDIR/../../../../scripts/diff_jsons.py" "$TESTDIR/../data/dataset5.json" dataset.json \
```

Comment:
> I found [pytest tests](https://github.com/nextstrain/augur/blob/master/tests/util_support/test_auspice_config.py) easier to understand for the config parsing, but the tests added here look ok too!

Ah good to know/remember that there are pytest tests for the Auspice config parsing! In this particular case I wanted to check the exported Auspice dataset has matching keys for the colorings/filters configs and the `node_attrs` so it made more sense as a functional test.